otel-extensions/src/main/java/org/hypertrace/agent/otel/extensions/config/HypertraceConfig.java

```diff
@@ -189,4 +219,27 @@ private static String convertYamlToJson(InputStream yaml) throws IOException {
     ObjectMapper jsonWriter = new ObjectMapper();
     return jsonWriter.writeValueAsString(obj);
   }
+
+  /**
+   * Creates a collection of objects consisting of the set of objects specified in the
```

Comment:
Should we have the config allowed content types just do the override? i.e if you set the config yourself, you need to add the defaults as well in the config i.e

Reply:
yep

Reply:
ok, I wasn't sure what was the right thing to do here.  Will fix